tests/e2e/taqueria-plugin-taquito.spec.ts

```diff
@@ -0,0 +1,346 @@
+        }
+
+    });
+
+    test('Verify that taqueria taquito plugin can deploy multiple contracts using deploy command', async () => {
+        try {
+            environment = "test";
+            let smartContractOneHash = "";
+            let smartContractTwoHash = "";
+
+            // 1. Copy config.json and two michelson contracts from data folder to artifacts folder under taqueria project
+            execSync(`cp e2e/data/config-taquito-test-environment-multiple-contracts.json ${taqueriaProjectPath}/.taq/config.json`);
+            execSync(`cp e2e/data/hello-tacos.tz ${taqueriaProjectPath}/artifacts/hello-tacos-one.tz`);
+            execSync(`cp e2e/data/hello-tacos.tz ${taqueriaProjectPath}/artifacts/hello-tacos-two.tz`);
+
+            // Sometimes running two deploy commands in a short period of time cause an issue
+            // More details about the error - "counter %x already used for contract %y"
+            // It is applicable only for auto-tests that run very fast
+            // Uses retry mechanism to avoid test to fail
+            await waitForExpect(() => {
+                // 2. Run taq deploy on a network described in "test" environment
+                const stdoutDeploy = execSync(`taq deploy -e ${environment}`, {cwd: `./${taqueriaProjectPath}`}).toString().trim().split(/\r?\n/);
+
+                // 3. Verify that contracts have been originated on the network
+                expect(stdoutDeploy[3]).toContain("hello-tacos-one.tz");
+                expect(stdoutDeploy[3]).toContain("hangzhounet");
+                const [,,contractOneHash] = stdoutDeploy[3].split("│");
+                smartContractOneHash = contractOneHash.trim();
+                expect(smartContractOneHash).toMatch(contractRegex);
+                expect(stdoutDeploy[5]).toContain("hello-tacos-two.tz");
+                expect(stdoutDeploy[5]).toContain("hangzhounet");
+                const [,,contractTwoHash] = stdoutDeploy[5].split("│");
+                smartContractTwoHash = contractTwoHash.trim();
+                expect(contractTwoHash.trim()).toMatch(contractRegex);
+            });
+
+            // 4. Verify that contracts have been originated to the network
+            await waitForExpect(async () => {
+                const contractOne = await tezos.contract.at(smartContractOneHash);
+                expect(contractOne.address).toBe(smartContractOneHash);
+                const contractTwo = await tezos.contract.at(smartContractTwoHash);
+                expect(contractTwo.address).toBe(smartContractTwoHash);
+            });
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+
+    });
+
+    // TODO: Currently there is no specific message for this type of error
+    // Issue associated https://github.com/ecadlabs/taqueria/issues/313
+    test.skip('Verify that taqueria taquito plugin will show proper error when environment does not exists', async () => {
```

Comment:
Ivan, do you mind creating an issue to restore this test case once #313 is resolved, and add the link to that issue as a TODO comment here. In the issue description, you can note that the issue is blocked by #313 which will link the two issues together.

Reply:
Btw, I think this particular problem is resolved in the upcoming v0.0.9 release.

Reply:
I checked 0.1.1 and the issue is still there. 
![Greenshot 2022-02-24 07 41 49](https://user-images.githubusercontent.com/10706087/155557204-4f5fbcd0-0d8d-4692-967c-1f20e368150e.png)